src/arc.ts

```diff
@@ -101,11 +103,91 @@ export function providerHasConfigUi(provider?: any): any | undefined {
   return provider && provider.isTorus;
 }
 
+function getBiconomy(provider: any): any {
+  console.log("getting biconomy");
+  if (biconomy) {
+    return biconomy;
+  }
+  biconomy = new Biconomy(provider, {
+    dappId: process.env.BICONOMY_DAPP_ID,
+    apiKey: process.env.BICONOMY_API_KEY,
+    debug: true,
+  });
+  return biconomy;
+}
+
+function showUserNotification(message: string, options?: IEnableWalletProviderParams, type?: NotificationStatus) {
+  if (options && options.showNotification && type)
+    options.showNotification(type, message);
+  else
+    alert(message);
```

Comment:
All alerts should be removed.  If a notification is absolutely desired, please use `showNotification`, but limit notifications to those that are truly needed.  If we must show modal information that requires user interaction (like clicking an "Ok" button) then we need to use `ModalPopup`, and ideally create a nice reusable component for that.

Reply:
Can you help implementing this ModalPopup in arc.ts?

Reply:
Sure, leave it as an alert for now.

Reply:
The PR on this is here: https://github.com/orenyodfat/alchemy/pull/2